### ISSUE TITLE
Use "group" icon in group PM avatar 

### ISSUE THREAD

**Issue by alya** [zulip/zulip-mobile#4870]
At present, the group PM avatar shows first initials of the participants, which doesn't look great and doesn't actually make it easy to figure out at a glance who is in the conversation.

Instead of initials, we should use a simple "group" icon, similar to the huddle icon we show in the left sidebar on desktop. To keep the groups visually distinct, we should continue to use random background colors for the group avatar.

**Comment by thisisnitish** [comment]
@alya I would like to work on this issue but I've some doubts such as what is PM avatar. I've found one group PM when you send personal msg to someone. Is this the same PM are this issue about?

**Comment by alya** [comment]
Hi @thisisnitish ! If you start a group PM thread on your Zulip account, you can see how group PMs are displayed when you click on the people icon in the bottom navigation, which takes you to all the recent conversations.

**Comment by thisisnitish** [comment]
Hi @alya I think you are talking about this?
![Simulator Screen Shot - iPhone 11 - 2021-07-23 at 22 50 37](https://user-images.githubusercontent.com/48220369/126819120-686f221e-aedf-4008-a034-ea637eca7fdc.png)

**Comment by thisisnitish** [comment]
Hi @alya I understood the problem, the avatar is taking the first initials of the members.

**Comment by alya** [comment]
Right, and instead of the icon with first initials, we just want to use the
same people icon as in the webapp (https://fontawesome.com/v4.7/icon/users).

**Comment by thisisnitish** [comment]
Sure @alya, I'll try to fix this issue as soon as possible 🙂👍🏻
